ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AvailableStorageTable.tsx

```diff
@@ -667,6 +667,7 @@ const AvailableStorageTable = ({
               content: (
                 <div className="u-flex">
                   <GroupCheckbox
+                    disabled={actionsDisabled}
```

Comment:
Maybe there should be a test for this so we can catch it if this somehow breaks again.

Reply:
Yep, great idea